### PR TITLE
Add configs for Pattern-3: Simple scalable deployment

### PR DESCRIPTION
**Pull request by RakhithaRR**
## Purpose
This PR adds the initial k8s and helm configurations for the new Pattern-3 [Simple scalable deployment](https://apim.docs.wso2.com/en/latest/install-and-setup/setup/deployment-overview/#simple-scalable-deployment).

### CODE REVIEW

File: advanced/am-pattern-3/templates/am/control-plane/instance-1/wso2am-pattern-3-am-control-plane-service.yaml [hunk 1]
```diff
@@ -15,24 +15,24 @@
...
       protocol: TCP
-      targetPort: 9711
+      port: 9711
     - name: jms-tcp
+      protocol: TCP
       port: 5672
+    - name: servlet-https
       protocol: TCP
-      targetPort: 5672
+      port: 9443
```
Comment (sajithaliyanage) [inline advanced/am-pattern-3/templates/am/control-plane/instance-1/wso2am-pattern-3-am-control-plane-service.yaml:38]: Shall we add a new line at EOF?

File: advanced/am-pattern-3/values.yaml [hunk 2]
```diff
@@ -126,190 +129,115 @@ wso2:
...
 
-      # API Manager's DevPortal specific configurations
-      devportal:
         # Configure Ingress
         ingress:
-          # Hostname for DevPortal profile
-          hostname: "devportal.am.wso2.com"
-          # Annotations for the API Manager DevPortal service Ingress
+          # Hostname for Gateway profile
+          hostname: "am.wso2.com"
```
Comment (sajithaliyanage) [inline advanced/am-pattern-3/values.yaml:150]: "am.wso2.com" or "gateway.am.wso2.com"?

File: advanced/am-pattern-3/templates/am/control-plane/instance-2/wso2am-pattern-3-am-control-plane-service.yaml [hunk 3]
```diff
@@ -15,18 +15,24 @@
...
+      port: 9611
+    - name: binary-secure
       protocol: TCP
+      port: 9711
+    - name: jms-tcp
+      protocol: TCP
+      port: 5672
+    - name: servlet-https
+      protocol: TCP
+      port: 9443
```
Comment (sajithaliyanage) [inline advanced/am-pattern-3/templates/am/control-plane/instance-2/wso2am-pattern-3-am-control-plane-service.yaml:38]: Shall we add a new line here?

**Review by sajithaliyanage — APPROVED** [review]

File: advanced/am-pattern-3/templates/am/control-plane/instance-1/wso2am-pattern-3-am-control-plane-service.yaml (hunk 1 above)
Comment (RakhithaRR) [reply advanced/am-pattern-3/templates/am/control-plane/instance-1/wso2am-pattern-3-am-control-plane-service.yaml]: Fixed with 2188616

File: advanced/am-pattern-3/values.yaml (hunk 2 above)
Comment (RakhithaRR) [reply advanced/am-pattern-3/values.yaml:150]: Fixed with 2188616

File: advanced/am-pattern-3/templates/am/control-plane/instance-2/wso2am-pattern-3-am-control-plane-service.yaml (hunk 3 above)
Comment (RakhithaRR) [reply advanced/am-pattern-3/templates/am/control-plane/instance-2/wso2am-pattern-3-am-control-plane-service.yaml]: Fixed with 2188616

**Comment by sajithaliyanage** [comment]
Fixes: https://github.com/wso2/kubernetes-apim/issues/499

**Pull request merged**